app/lib/utils.py

```diff
@@ -64,6 +64,9 @@ def fetch_json(remote_url, method='GET', data=None, params=None, headers=None):
     if method == "DELETE":
         return True
 
+    if r.status_code == 204:
```

Comment:
Why should we put the `status_code` checking here? I think it better to be in `fetch_remote` function, no?